10/umbraco-cms/fundamentals/setup/upgrading/version-specific/minor-upgrades-for-umbraco-7.md

```diff
@@ -0,0 +1,103 @@
+
+You can expect some changes to the following configuration files:
+
+* Any file in the `/Config` folder
+* The `/Global.asax` file
+* The `web.config` file in the root of your site **(Important: make sure to copy back the version number, and the connection string as they were.)**
+* In rare cases, the `web.config` file in the Views folder
+
+Use a tool like [WinMerge](http://winmerge.org/ "WinMerge") to check changes between all of the config files. Depending on when you last did this there may have been updates to quite a few of them.
+
+There's also the possibility that some files in the `/Config` folder are new or some have been removed (we do make a note of this in the release notes). WinMerge (and other diff tools) is able to compare folders as well so you can spot these differences.
+
+Up until version 6.0.0 it was necessary to change the version number in `ClientDependency.config`. This was to clear the cached html/css/js files in the backoffice. Change the current version number to one that's higher than that. Make sure not to skip this step as you might get strange behaviour in the backoffice otherwise.
+
+## Merge UI.xml and language
+
+Some packages (like Contour and Umbraco Forms) add dialogs to the `UI.xml`. Make sure to merge those changes back in from your backup during the upgrade so that the packages continue to work. This file can be found in: `/Umbraco/Config/Create/UI.xml`.
+
+Packages like Contour, Umbraco Forms, and Courier also make changes to the language files located in: `/Umbraco/Config/Lang/*.xml` (typically `en.xml`).
+
+## Finalize
+
+After copying the files and making the config changes, you can open your site. You should see the installer which will guide you through the upgrade.
+
+The installer will do two things:
+
+* Update the version number in the `web.config`
+* Upgrade your database in case there are any changes
+
+We are aware that, currently, the installer is asking you for the database details of a **blank database** while upgrading. In the near future this will be pre-filled with your existing details and the wording will be updated. So no need to be scared. Enter the details of your existing database and Umbraco will upgrade it to the latest version when necessary.
+
+## Post installation
+
+One important recommendation is to always remove the `install` folder immediately after upgrading Umbraco and never to upload it to a live server.
+
+## Potential issues and gotchas
+
+### Browser cache
+
+Google Chrome has notoriously aggressive caching, so if something doesn't seem to work well in the backoffice, make sure to clear cache and cookies thoroughly (for other browsers as well). Normally the browser cache problem is automatically handled in an Umbraco upgrade by modifying the config/ClientDependency.config version number. If you however wish to re-force this update you can increment this version number which will ensure that any server-side cache of JavaScript and stylesheets gets cleared as well.
```

Comment:
```suggestion
Google Chrome has notoriously aggressive caching. If something doesn't seem to work well in the backoffice, make sure to clear cache and cookies thoroughly (for other browsers as well). Normally the browser cache problem is automatically handled in an Umbraco upgrade by modifying the config/ClientDependency.config version number. If you wish to re-force this update you can increment this version number. This will ensure that any server-side cache of JavaScript and stylesheets gets cleared as well.
```